plugins/include/engine.inc

```diff
@@ -24,242 +24,1271 @@
+ */
 native get_global_int(variable);
+
+/**
+ * Retrieves a global string type value from the server.
+ *
+ * @note For a list of valid string type entries, see the GL_* constants in
+ *       engine_const.inc under the "String" section.
+ *
+ * @param variable  Entry to retrieve from
+ * @param string    Buffer to copy value to
+ * @param maxlen    Maximum size of buffer
+ *
+ * @return          Number of cells written to buffer
+ * @error           If an invalid entry is provided, an error will be thrown.
+ */
 native get_global_string(variable, string[], maxlen);
+
+/**
+ * Returns a vector type value from the server globals.
+ *
+ * @note For a list of valid vector type entries, see the GL_* constants in
+ *       engine_const.inc under the "Vector" section.
+ *
+ * @param variable  Entry to retrieve from
+ * @param vector    Array to store vector in
+ *
+ * @noreturn
+ * @error           If an invalid entry is provided, an error will be thrown.
+ */
 native get_global_vector(variable, Float:vector[3]);
+
+/**
+ * Returns a edict type value from the server globals.
+ *
+ * @note For a list of valid edict type entries, see the GL_* constants in
+ *       engine_const.inc under the "Edict" section.
+ * @note This native returns 0 as an error value if the edict retrieved is an
+ *       invalid entity. As 0 is an entity index that is considered to be a
+ *       valid value for some globals ("worldspawn"), this native can
+ *       potentially return a misleading value. Use get_global_edict2() for a
+ *       safe version.
+ *
+ * @param variable  Entry to retrieve from
+ *
+ * @return          Value of specified entry
+ * @error           If an invalid entry is provided, an error will be thrown.
+ */
 native get_global_edict(variable);
 
-/* Set entity bounds. */
+/**
+ * Sets the size of the entity bounding box, as described by the minimum and
+ * maximum vectors relative to the origin.
+ *
+ * @param index     Entity index
+ * @param mins      Vector containing the minimum point relative to the origin
+ * @param maxs      Vector containing the maximum point relative to the origin
+ *
+ * @noreturn
+ * @error           If an invalid entity index is provided, an error will be
+ *                  thrown.
+ */
 native entity_set_size(index, const Float:mins[3], const Float:maxs[3]);
 
-/* Get decal index */
+/**
+ * Returns the index of a decal.
+ *
+ * @param szDecalName   Decal name
+ *
+ * @return              Decal index >= 0, or -1 if decal was not found
+ */
 native get_decal_index(const szDecalName[]);
 
-/* Returns the distance between two entities. */
-native Float:entity_range(ida,idb);
+/**
+ * Returns the distance between two entities.
+ *
+ * @param ida   Entity index 1
+ * @param idb   Entity index 2
+ *
+ * @return      Distance between the entities
+ * @error       If an invalid entity index is provided or, if the index is a
+ *              client index, the client is not connected, an error will be
```

Comment:
You can, it is not specific either way. When it says "entity" it means literally any entity. If functions are specific to clients it will explicitly say so.